src/component/InspirationImg/InspirationImg.js

```diff
@@ -0,0 +1,34 @@
+import {Carousel} from 'react-responsive-carousel';
+import 'react-responsive-carousel/lib/styles/carousel.min.css';
+import styled from 'styled-components';
+
+export default function InspirationImg({inspirationImg1, inspirationImg2, inspirationImg3}) {
+  return (
+    <ImageContainer>
+      <Carousel showStatus={false} centerMode={true} width="250px" showThumbs={false} >
+        <img src={inspirationImg1} alt="first inspiration" style={{border: 'black solid 3px'}} />
+        <img src={inspirationImg2} alt="second inspiration" style={{border: 'black solid 3px'}} />
+        <img src={inspirationImg3} alt="third inspiration" style={{border: 'black solid 3px'}} />
+      </Carousel>
+    </ImageContainer>
+  );
+}
+
+const ImageContainer = styled.div`
+  display: flex;
+  margin: auto;
+  max-width: 340px;
+  overflow: hidden;
```

Comment:
I guess you don't need your `overflow: hidden` because the library provides you with this property for the carousel.

Reply:
thanks for the hint. I delete the overflow